src/vsr/README.md

```diff
@@ -1,3 +1,17 @@
+# Glossary
+
+- _checkpoint_: Ensure that all updates from the past wrap of the WAL are durable in the _grid_, then advance the replica's recovery point by updating the superblock. After a checkpoint, the checkpointed WAL entries are safe to be overwritten by the next wrap. (Sidenote: in consensus literature this is sometimes called snapshotting. But we use that term to mean something else.)
+- _grid_: The zone on disk where LSM trees and metadata for them reside.
+- _header_: Identifier for an entry in the VSR log. Passed around instead of the entry when the full entry is not needed (such as view change).
```

Comment:
_header_ can be this specific (log entry) in context, but it is more general (all messages/blocks have a header).

Reply:
How about

> Identifier for many kinds of messages, including each entry in the VSR log. Passed around instead of the entry when the full entry is not needed (such as view change).

Reply:
Yeah, that works